### PR TITLE
Open image planes as arrays

### PR DESCRIPTION
**Pull request by simleo**
Adds a Python-level `OMETIFFReader` which adds an `open_array` method to return image planes as NumPy arrays. Updates examples to use the new method.

### CODE REVIEW

File: examples/dump_planes.py [hunk 1]
```diff
@@ -29,25 +29,11 @@ def dump(ome_tiff_fn):
     print "image size: %d x %d" % (W, H)
     print "pixel type:", dtype
     for i in xrange(img_count):
```
Comment (dgault) [inline examples/dump_planes.py]: Can some of the above values be removed now that they are included as part of the new open_array method? (H, W, dtype, interleaved) Or is it still useful to keep them for the purposes of the print statements?

Comment (simleo) [inline examples/dump_planes.py]: Yes, I thought about removing them as well, but I wanted to keep the printed info.

Comment (dgault) [inline examples/dump_planes.py]: Makes sense, I can see it being useful to have the info printed here

**Comment by dgault** [comment]
Built and tested the dump_planes.py with a number of different OME-TIFF files, output displayed was as expected. All planes were written correctly and opened and displayed without issue.

PR looks good to merge.

**Pull request merged**